source/guide/3.2/guide-policies.md

```diff
@@ -94,11 +94,88 @@ Built-in policies are declared in [`types.yaml`]({{page.types_yaml_link}}), whic
+
+    cloudify.policies.types.threshold:
+        properties: &THRESHOLD_BASED_POLICY_PROPERTIES
+            <<: *BASIC_AH_POLICY_PROPERTIES
+            service:
+                description: The service name
+                default: service
+            threshold:
+                description: The metric threshold value
+            upper_bound:
+                description: |
+                    boolean value for describing the semantics of the threshold.
+                    if 'true': metrics whose value is bigger than the threshold will cause the triggers to be processed.
+                    if 'false': metrics with values lower than the threshold will do so.
+                default: true
+            stability_time:
+                description: How long a threshold must be breached before the triggers will be processed
+                default: 0
+        source: https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/resources/rest-service/cloudify/policies/threshold.clj
+
+    cloudify.policies.types.ewma_stabilized:
+        properties:
+            <<: *THRESHOLD_BASED_POLICY_PROPERTIES
+            ewma_timeless_r:
+                description: |
+                    r is the ratio between successive events. The smaller it is, the smaller impact on the computed value the most recent event has.
+                default: 0.5
+        source: https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/resources/rest-service/cloudify/policies/ewma_stabilized.clj
 
 {% endhighlight %}
 
+* Host failure policy
+
+    This policy is based on intercepting expired events. The first monitoring event sent by a certain node instance will add this event type (determined by the event's service and host properties) to the policy engine index (Riemann's index mechanism). Later, if this type of event does not get sent for a period of 60 seconds for this particular node instance, it gets expired.
+
+    When an event expires and has been sent by a service contained in the list of services (policy's property) specified in the blueprint, the host failure policy passes this event to the restraints check and eventually processes triggers. It adds the `diagnose` field with value "heart-beat-failure" and the `failing_node` field with an id of the failing node to the event. Additionally, the event's state gets changed to "triggering_state".
+
+    This policy's implementation is based on [expired? Riemann function](http://riemann.io/api/riemann.streams.html#var-expired.3F).
+
+    You can find it on [github](https://github.com/cloudify-cosmo/cloudify-manager/blob/master/resources/rest-service/cloudify/policies/host_failure.clj).
+
+* Threshold policy
+
+    When for `stability_time` seconds all not expired events have metric that breach the `threshold`, the policy passes the last event to the restraints check and eventually processes triggers.
```

Comment:
When for stability_time seconds all not expired events have metric that breach the threshold, the policy passes the last event to the restraints check and eventually processes triggers. -->
When for `stability_time` seconds, all none expired events have a `metric` value that breaches the threshold, the policy passes the last event to the restraints check and eventually processes the policy triggers.